src/user/dashboard/updates/updates.scss

```diff
@@ -1,33 +1,30 @@
 .updates {
   min-height: 300px;
   height: 400px;
-  margin: 20px 0;
-  overflow: auto;
+  margin: 30px 0 0 40px;
   h1 {
-    font-size: 26px;
+    font-size: 28px;
     text-align: center;
     font-weight: bold;
   }
   .org-updates {
```

Comment:
increase the height of org updates in pinned post section so that it occupy whole left sidebar.

Reply:
I have done this in commit f276d1a.